.github/actions/api-deploy-ecs/action.yml

```diff
@@ -72,6 +72,7 @@ runs:
         run: |
           mv ./flagsmith-saml/saml ./api
           cat ./flagsmith-saml/requirements.txt >> ./api/requirements.txt
+          echo "" >> ./api/requirements.txt
```

Comment:
Might be worth adding a comment here to explain why we're doing this? 

```yaml
echo "" >> ./api/requirements.txt  # add a new line so that future concatenation works
```

Reply:
Yeah, good point